README.md

```diff
@@ -59,6 +59,9 @@ yarn dev
 
 You should receive an Etherscan link with your sponsored transaction in the terminal output. [Example](https://sepolia.basescan.org/tx/0xe51e9bf6fea0dfecfcbf7168bcc7da2c833ad0dcac5651940953a89857674885)
 
+### 7. Play around with our demo app
```

Comment:
can you move this below the code block

<img width="880" alt="image" src="https://github.com/coinbase/paymaster-bundler-examples/assets/93971719/ff1edb38-c561-431c-ad8c-88c776a44a36">